src/pages/components/pane.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Pane
+description: >
+  Panes give users a quick way to resize layouts vertically or horizontally by
+  interacting with the splitter. They can also be used to create expandable or
+  collapsible layouts.
+package: '@zendeskgarden/react-grid'
+components:
+  - grid/src/elements/pane/PaneProvider.tsx
+  - grid/src/elements/pane/Pane.tsx
+subcomponents:
+  - grid/src/elements/pane/components/Content.tsx
+  - grid/src/elements/pane/components/Splitter.tsx
+  - grid/src/elements/pane/components/SplitterButton.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To make a structured layout resizable, expandable or collapsible</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Vertical
+
+Panes can be used in a vertical orientation. The panes resize up to the values
+that you set.
+
+import PaneVertical from '../../examples/pane/PaneVertical.tsx';
+import PaneVerticalCode from '!!raw-loader!../../examples/pane/PaneVertical.tsx';
+
+<CodeExample code={PaneVerticalCode}>
+  <PaneVertical />
+</CodeExample>
+
+### Horizontal
+
+Panes can be used in a horizontal orientation as well.
+
+import PaneHorizontal from '../../examples/pane/PaneHorizontal.tsx';
+import PaneHorizontalCode from '!!raw-loader!../../examples/pane/PaneHorizontal.tsx';
+
+<CodeExample code={PaneHorizontalCode}>
+  <PaneHorizontal />
+</CodeExample>
+
+### Grid
+
+Panes can be resized in both vertical and horizontal directions for a complex
+grid flexible layout.
+
+import PaneGrid from '../../examples/pane/PaneGrid.tsx';
+import PaneGridCode from '!!raw-loader!../../examples/pane/PaneGrid.tsx';
+
+<CodeExample code={PaneGridCode}>
+  <PaneGrid />
+</CodeExample>
+
+## Customization
+
+### Vertical collapse
+
+Panes can be collapsed or expanded by using the splitter together with an
+[Icon button](/components/icon-button). Vertical orientation uses a small circular
+icon button positioned on top of the splitter by default in both collapsed and
+expanded states. [Tooltips](/components/tooltip) should be used to describe the action.
+
+import PaneVerticalCollapse from '../../examples/pane/PaneVerticalCollapse.tsx';
+import PaneVerticalCollapseCode from '!!raw-loader!../../examples/pane/PaneVerticalCollapse.tsx';
+
+<CodeExample code={PaneVerticalCollapseCode}>
+  <PaneVerticalCollapse />
+</CodeExample>
+
+### Horizontal collapse
+
+Horizontal orientation also uses a small [Icon button](/components/icon-button) to
+collapse the pane. When the pane is collapsed, a stretched [Button](/components/button)
+is used to indicate an expandable state of the pane.
```

Comment:
Ditto. These component link-outs are not technically valid. The eventual implementation of the horizontal action will _not_ be explicitly based on `<Button isStretched>`.

Reply:
I think we can remove the second sentence as we don't have that functionality yet and rework in the future. The first sentence can be changed to match the discussion above.

Reply:
According to @smritimadan 's original post in the PR, horizontal collapse is not implemented yet. 

My take is that we remove this section for now and update to include it when it's implemented. @lucijanblagonic We should rework this section at that time.

Reply:
Quick clarification @diana-oum , the standard view for the horizontal collapse (just the centered SplitterButton) is already implemented. The _alternate view_ for the horizontal collapse (stretched button) is not implemented yet.

Reply:
@smritimadan Ah I see. Thanks for clarifying

Reply:
Agree with @lucijanblagonic, that we should rework in the future when we implement the alternate view for horizontal collapse. In continuation of the conversation in the above [comment](https://github.com/zendeskgarden/website/pull/416#discussion_r938272689), okay to remove this whole section for this PR?